### DISCUSSION TITLE
Error during simulation

### DISCUSSION THREAD

**Discussion by AngelicaScamarcia** [sofa-framework/sofa#3725] [Build / Config / Environment] [answered]
Hi there, 
I have an old project and I'm trying to use it in the new version of Sofa but I'm facing this problem when I run the simulation in Sofa:

[ERROR]   [DefaultContactManager] Option "FrictionContact" for data "response" has been renamed since v21.12 (PR#2522). Use "FrictionContactConstraint" instead.

Someone knows how can I fix it?

**Comment by alxbilger** [comment, marked as answer]
Duplicate of https://github.com/sofa-framework/sofa/discussions/3721
